data/data/bootstrap/files/usr/local/bin/release-image-download.sh.template

```diff
@@ -17,3 +17,20 @@ do
     echo "Pull failed. Retrying $RELEASE_IMAGE..."
 done
 
+{{if .IsOKD -}}
+# Pivot bootstrap to FCOS + OKD machine-os-content
```

Comment:
The release-image-download unit does not seem like the appropriate place to be doing the pivot.

Reply:
I can't seem to find a better place for it - any suggestions? Should we have an additional `release-image-pivot.service`?

Reply:
Yeah, I think it should be its own service that is only enabled for OKD.

Reply:
Done